syntax = "proto3";
package micro_app;




import "model/app_store/app_version.proto";

/*
GetAppStoreMicroApp请求
*/
message GetAppStoreMicroAppRequest {
    /*
    小产品id
    */
    string app_id = 1 ;
}

/*
GetAppStoreMicroApp返回
*/
message GetAppStoreMicroAppResponse {
    /*
    当前版本
    */
    app_store.AppVersion currentVersion = 1 ;
    /*
    小产品名称
    */
    string name = 2 ;
    /*
    小产品id
    */
    string id = 3 ;
    /*
    图标url
    */
    string icon = 4 ;
    /*
    小产品概要介绍
    */
    string intro = 5 ;
    /*
    功能预览
    */
    repeated string preview = 6 ;
    /*
    功能详细介绍
    */
    string description = 7 ;
}

/*
GetAppStoreMicroAppApi返回
*/
message GetAppStoreMicroAppResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    micro_app.GetAppStoreMicroAppResponse data = 4 ;
}

